src/main/java/com/i6/honterview/controller/FileController.java

```diff
@@ -0,0 +1,73 @@
+package com.i6.honterview.controller;
+
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+
+import org.springframework.core.io.Resource;
+import org.springframework.http.ContentDisposition;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import com.i6.honterview.dto.request.FileUploadRequest;
+import com.i6.honterview.dto.response.FileUploadResponse;
+import com.i6.honterview.response.ApiResponse;
+import com.i6.honterview.service.FileService;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.RequiredArgsConstructor;
+
+@Tag(name = "파일")
+@RestController
+@RequestMapping("/api/v1/files")
+@RequiredArgsConstructor
+public class FileController {
+
+	private final FileService fileService;
+
+	@Operation(summary = "파일 업로드", description = "파일을 S3에 업로드합니다.")
+	@PostMapping("/upload")
+	public ResponseEntity<ApiResponse<FileUploadResponse>> uploadFile( // TODO : answer와 연결
+		@RequestPart("request") FileUploadRequest request,
+		@RequestPart("file") MultipartFile file
+	) {
```

Comment:
```suggestion
	@Operation(summary = "파일 업로드", description = "파일을 S3에 업로드합니다.")
	@PostMapping(path = "/upload", consumes = MediaType.MULTIPART_FORM_DATA_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
	public ResponseEntity<ApiResponse<FileUploadResponse>> uploadFile( // TODO : answer와 연결
		@RequestPart("request") @Parameter(schema = @Schema(type = "string", format = "binary")) FileUploadRequest request,
		@RequestPart("file") MultipartFile file
	) {
```
swagger 문서에서도 테스트 할 수 있도록 수정 부탁드려요!

Reply:
넵 감사합니다 ☺️